modules/installation-obtaining-installer.adoc

```diff
@@ -95,6 +95,11 @@ cluster, even if the cluster failed during installation. To remove your cluster,
 {product-title} uninstallation procedures for your specific cloud
 provider.
 ====
++
+[IMPORTANT]
+====
+Ensure that you download the OpenShift installer that corresponds with your host operating system and architecture.
```

Comment:
I think the wording is good and specifying the architecture makes sense with the AWS ARM scenario. My only recommendation would be to possibly move this info into the step 3 wording. What about something like this?
```
3. Navigate to the page for your installation type, download the installation program that
corresponds to your host operating system and architecture, and place the file in the directory
where you will store the installation configuration files.
```
Stacking three important admonitions together isn't best practice. I think having the actual step description mention all requirements would be ideal. Feel free to wordsmith that suggestion to make it even more clear :slightly_smiling_face:

Reply:
Yeah! Thats a great idea, and makes more sense.